### PR TITLE
refactor: Migrate `ResizeObserver` to composition API (no-changelog)

### PR DESCRIPTION
**Pull request by tomi**
## Summary

Migrate `ResizeObserver` to composition API 

## Review / Merge checklist

- [ ] PR title and summary are descriptive. ([conventions](../blob/master/.github/pull_request_title_conventions.md)) 
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [ ] Tests included. 
- [ ] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

### CODE REVIEW

File: packages/design-system/src/components/ResizeObserver/ResizeObserver.vue [hunk 1]
```diff
@@ -1,86 +1,63 @@
...
+const root = ref<HTMLDivElement | null>(null);
+
+const sortedBreakpoints = computed(() => [...props.breakpoints].sort((a, b) => a.width - b.width));
 
-		const root = this.$refs.root as HTMLDivElement;
+const getBreakpointFromWidth = (width: number): string => {
+	return (
+		sortedBreakpoints.value.find((sortedBreakpoint) => width < sortedBreakpoint.width)?.bp ??
+		'default'
+	);
```
Comment (alexgrozav) [inline packages/design-system/src/components/ResizeObserver/ResizeObserver.vue:32]: Shouldn't this 'default' be outside the `find`?

**Review by alexgrozav — APPROVED** [review]
LGTM ✅

**Comment by janober** [comment]
Got released with [`n8n@1.47.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.47.0)

**Comment by janober** [comment]
Got released with [`n8n@1.47.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.47.0)

**Pull request merged**